Chinese/deserthell/OldStory/Cat2Dog/saying/说了什么.md

```diff
@@ -0,0 +1,2 @@
+﻿白胡子老头说:“如果不是看在你爸的份儿上，今天我是不会再帮你的。快！原地转三圈！[然后......](./Cat2Dog/saying/说了什么.md)”
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.